FirebaseAuth/Sources/SystemService/FIRSecureTokenService.m

```diff
@@ -111,6 +111,7 @@ - (void)fetchAccessTokenForcingRefresh:(BOOL)forceRefresh
       complete();
       callback(self->_accessToken, nil, NO);
     } else {
+      NSLog(@"Fetching new token from backend.");
```

Comment:
NSLog will log messages unconditionally which may be undesirable for the app developers not interested in the messages. We have a [logger interface](https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseCore/Sources/Private/FIRLogger.h) with log levels which helps us to avoid too noisy logs produced by Firebase SDKs. Messages logged with `FIRLogDebug` will be visible only if debug logging is explicitly enabled by the developer. See usage example [here](https://github.com/firebase/firebase-ios-sdk/blob/8d9243454b0d141f520ba7495836d4be71b44a3d/FirebaseInstallations/Source/Library/InstallationsIDController/FIRInstallationsIDController.m#L159).

Reply:
Done.